protected/views/patient/_patient_all_episodes.php

```diff
@@ -32,7 +32,7 @@
 			<tr>
 				<th>Start Date</th>
 				<th>End Date</th>
-				<th><?php echo Firm::contextLabel(); ?></th>
+				<th>Service</th>
```

Comment:
Is there a serviceLabel function you could call instead of hardcoding the text here?